src/data/historikk/types/historikkTypes.ts

```diff
@@ -9,7 +9,8 @@ export type HistorikkEventType =
   | "VEILEDER_TILDELING"
   | "DIALOG_MED_BEHANDLER"
   | "SEN_OPPFOLGING"
-  | "DIALOGMOTEKANDIDAT";
+  | "DIALOGMOTEKANDIDAT"
+  | "OPPFOLGINGSPLAN_LPS";
```

Comment:
Burde vi hatt `OPPFOLGINGSPLAN` her også for completeness? Litt rart at den ikke er i lista, når vi har en tag for den i switch-casen.

Reply:
Den er inkludert i `HistorikkEventType` gjennom `HistorikkKilde`. Men det går sikkert an å refaktorere bort `HistorikkKilde` nå, virker litt unødvendig :thinking: